utils/helpers.py

```diff
@@ -43,28 +49,41 @@ def display_skipped(result):
             print('{0}\t \t{1} \t{2}'.format(value[1], key, value[0]))
 
 
-class DependencyChecker():
-    def __init__(self, columns):
-        self.columns = columns
+def is_valid_url(url):
+    """
+    checks if url is valid
+    :param url: url
+    :return: boolean
+    """
+    parsed_url = urllib.parse.urlparse(url)
+
+    if parsed_url.scheme and parsed_url.netloc:
+        return True
 
-    @classmethod
-    def check_dependency(cls, column, columns):
-        instance = cls(columns)
-        return instance.get_dependency(column)
+    print('Please enter a valid url.')
+    return False
 
-    @classmethod
-    def has_dep(cls, column, columns):
-        instance = cls(columns)
-        return instance.has_dep(column)
 
-    # def has_dep(self, column):
-    #     if self.columns[0] == column:
-    #         return False
-    #     return True
+def get_csv_from_url(url, filepath):
+    """
+    Get csv from URL
+    :param url: url that points to csv file
+    :param filepath: base path for csv files
+    :return: None or file object
+    """
 
-    def get_dependency(self, column):
-        if column in self.columns:
-            colunm_index = self.columns.index(column)
-            if colunm_index > 0:
-                return self.columns[colunm_index - 1]
+    filename = 'url_sample.csv'
```

Comment:
Instead of returning a hardcoded value for the filename for all csv files fetched over http, return the actual filename for the file that has been downloaded. That way, if say a user tries to seed data from multiple remote files a local copy of each is created without necessarily overwriting that one file (`url_sample.csv`).

Reply:
On it. Thanks for the feedback.